src/transformers/tokenization_utils_base.py

```diff
@@ -700,8 +700,11 @@ def convert_to_tensors(
                 raise ImportError("Unable to convert output to PyTorch tensors format, PyTorch is not installed.")
             import torch
 
-            as_tensor = torch.tensor
             is_tensor = torch.is_tensor
+
+            def as_tensor(value, dtype=None):
+                return torch.tensor(np.array(value))
```

Comment:
Could we just check here if the `value` is a list of `numpy.ndarray` since this is the case we have to fix, and simply use `torch.tensor` in the other cases?

Reply:
Nice suggestion!

Reply:
it's also works to list. 
```
import numpy as np
import torch
import random
import time

def measure():   
    randomlist = []
    for i in range(0, 1024000):
        a = random.randint(1,100)
        randomlist.append(a)
    # list -> tensor
    st = time.time()
    e = torch.tensor(randomlist)
    t3 = time.time() - st

    # list -> np -> tensor
    st = time.time()
    f = torch.tensor(np.array(randomlist))
    t4 = time.time() - st

    print(f"list->tensor: {t3} sec. | list->np->tensor: {t4} sec.")



for idx in range(10):
    measure()
```
results:
```
list->tensor: 0.11251711845397949 sec. | list->np->tensor: 0.08889651298522949 sec.
list->tensor: 0.11182880401611328 sec. | list->np->tensor: 0.056847572326660156 sec.
list->tensor: 0.11099457740783691 sec. | list->np->tensor: 0.05408644676208496 sec.
list->tensor: 0.11319661140441895 sec. | list->np->tensor: 0.052817344665527344 sec.
list->tensor: 0.11264157295227051 sec. | list->np->tensor: 0.05298781394958496 sec.
list->tensor: 0.10972905158996582 sec. | list->np->tensor: 0.05270671844482422 sec.
list->tensor: 0.10953164100646973 sec. | list->np->tensor: 0.05267834663391113 sec.
list->tensor: 0.10898041725158691 sec. | list->np->tensor: 0.053290367126464844 sec.
list->tensor: 0.10944771766662598 sec. | list->np->tensor: 0.05272650718688965 sec.
list->tensor: 0.11078786849975586 sec. | list->np->tensor: 0.0529322624206543 sec.
```

Reply:
actually. In my case,  i use tokenizer with padding is true. it will padding all the queries to the longest one.  i got almost 5ms latency reduced if one query is 800 Chinese characters.

Reply:
Yes, but you are not the only person using the Transformers library. For smaller sizes, it's faster to just call `torch.tensor` as is right now, so let's not change that part.

Reply:
i can fix it if you insist.  i just share my experience. not every one  have the chance or ability to make their model faster in several ms.